src/pages/components/form/usage.mdx

```diff
@@ -117,215 +115,378 @@ do you gain by collecting this information?
-  one box doesn’t uncheck the others. A stand-alone checkbox, or a toggle can be
-  used for a single option that the user can turn on or off.
-- **Select elements** are used for fields in which a single selection is
-  required and there are a large number of possible options.
+Optional helper text is pertinent information that assists the user in correctly
+completing a field. Helper text should always be available when the input is
+focused. It is often used to explain the correct data format. Helper text
+appears as persistent text under a fixed input (helper text is not available in
+fluid input, which uses tooltips instead). Helper text is the preferred method
+of assisting users because it is always visible for the user to reference
+without additional effort.
+
+#### Best practices for helper text
+
+- Use sentence-style capitalization, and in most cases, write the text as full
+  sentences with punctuation.
+- When used, helper text that appears persistently underneath the field is
+  replaced by an error or warning message if a state change occurs.
+
+### Tooltip help
+
+Tooltips can be added inline with a form label if additional non-pertinent
+information is need to assist the user in correctly completing a field, such as
+context setting. Tooltip content may include more in-depth information about the
+field or the options the user is choosing from. Links to additional information
+or references may also be include if needed. In certain situations where complex
+instructions or additional context would be appropriate, tooltips can be used in
+addition to the helper text. However, it is recommended that all pertinent
+assistive information to form completion—such as format content—be presented
+simply and directly as helper text under the field and not be in a tooltip
 
 <Row>
 <Column colLg={8}>
 
-![form logic](images/form-usage-4.png)
+![Example of text input with a tooltip](images/form-usage-6.png)
 
 </Column>
 </Row>
 
-## Validation and errors
+#### Tooltips in fluid components
 
-### Error messaging
+Fluid inputs are an exception to the no critial information in tooltips rule.
+Due to their architectural nature, fluid inputs do not provide the allotted
+space under the field needed for helper text. All assistive text wither
+pertinent or additional is provided through a tooltip when using fluid inputs.
 
-Effective error messaging can help the user to understand the problem and how to
-fix it. First, inform the user what has happened, then provide guidance on next
-steps or possible resolutions.
-[Inline notifications](/components/notification/usage) can be used to state the
-general problem with the users input, such as _“Please input the required
-fields.”_ Inline Notifications can occur pre or post submission, depending on
-the type of data the user is inputting.
+<Row>
+<Column colLg={8}>
+
+![Example of a fluid input using a tooltip for help](images/form-usage-7.png)
```

Comment:
This image has a `etc.` typo as well.
<img width="551" alt="Screen Shot 2023-01-24 at 1 01 41 PM" src="https://user-images.githubusercontent.com/43969356/214388924-00cffaa8-6660-46b8-8b6c-ea3cf02d598f.png">